documentation/AuthoringSolutionsTutorial.md

```diff
@@ -0,0 +1,342 @@
+## Authoring Solutions Registry Entries
+
+This tutorial will show how to write a new solution for integration with the GPII.  Solutions could be anything from an operating system accessibility feature such as screen magnification, or a third party program such as a screenreader.  For this tutorial we will use an example that was started during the Crete HCII 2014 workshops, integrating the windows based screen reader JAWS into the GPII. This example was meant as a tutorial, and needs more work to be production ready, but serves as a good example all the different parts that are needed.
+
+### Expressing Your Applications as a Document
+
+Integrating your solution, application, or other utility involves authoring a document using our structured solution schema formatted as JSON. In most cases you won't need to write any javascript code, as we have a number of plugins with syntax that are available from this schema. On occasion though, you may find something that can't be accomplished with the current schema and plugins, at which point you can work with our developers on creating the necessary extensions.
```

Comment:
Global replace javascript -> JavaScript

typo where to try it -> were to try it

which are different from -> to values different from the previous user's